bin/reversion.sh

```diff
@@ -43,4 +71,27 @@ fi
 
 echo "Changing from $VERSION to $NEWVERSION"
 
-find . -name 'pom.xml' -exec sed -i "s/<version>$VERSION<\/version>/<version>$NEWVERSION<\/version>/g" {} \;
+find . -name 'pom.xml' -exec sed -i "s/<version>$VERSION<\/version>/<version>$NEWVERSION<\/version>/g" {} \; -exec git add {} \;
+
+OLDSCHEMAMAJOR=`echo $VERSION|cut -d. -f1`
+OLDSCHEMAMINOR=`echo $VERSION|cut -d. -f2`
+OLDSCHEMAVERSION=`echo $VERSION|cut -d. -f1,2`
+NEWSCHEMAMAJOR=`echo $NEWVERSION|cut -d. -f1`
+NEWSCHEMAMINOR=`echo $NEWVERSION|cut -d. -f2`
+NEWSCHEMAVERSION=`echo $NEWVERSION|cut -d. -f1,2`
+
+if [ "$OLDSCHEMAVERSION" != "$NEWSCHEMAVERSION" ] && [ "$PROCESS_SCHEMAS" = true ] ;  then
+    echo "Current schema: $OLDSCHEMAVERSION"
+    echo "New schema:     $NEWSCHEMAVERSION"
+    # Create new unified configuration
+    cp core/src/test/resources/configs/unified/$OLDSCHEMAVERSION.xml core/src/test/resources/configs/unified/$NEWSCHEMAVERSION.xml
+    sed -i "s/:$OLDSCHEMAMAJOR.$OLDSCHEMAMINOR/:$NEWSCHEMAMAJOR.$NEWSCHEMAMINOR/g" core/src/test/resources/configs/unified/$NEWSCHEMAVERSION.xml
```

Comment:
This file still contains ``` xsi:schemaLocation="urn:infinispan:config:9.3 http://www.infinispan.org/schemas/infinispan-config-9.2.xsd"```

Reply:
(after replacement)

Reply:
There is maven goal to extract version components and expose them as variables, this could be used for those major.minor strings
http://www.mojohaus.org/build-helper-maven-plugin/parse-version-mojo.html